paig-server/backend/paig/api/governance/controllers/ai_app_config_download_controller.py

```diff
@@ -30,6 +33,7 @@ async def get_ai_app_config_json_data(self, id: int) -> Tuple[str, dict]:
         shield_server_key: EncryptionKeyView = await self.encryption_key_service.get_active_encryption_key_by_type(EncryptionKeyType.MSG_PROTECT_SHIELD)
         shield_plugin_key: EncryptionKeyView = await self.encryption_key_service.get_active_encryption_key_by_type(EncryptionKeyType.MSG_PROTECT_PLUGIN)
         shield_server_url = await self.ai_app_service.get_shield_server_url()
+        asyncio.create_task(capture_event_on_action(event=DownloadAIApplicationConfigEvent()))
```

Comment:
can we centralize code here instead of calling asyncio create task at every point ?